### PR TITLE
Update Ca/Cb_subset docs; Add get_opdm/get_tpdm docs.

### PR DESCRIPTION
**Pull request by JonathonMisiewicz**
Let's not have this burn anybody else.

## Todos
Notable points (developer or user-interest) that this PR has or will accomplish.
- [x] Documented gotchas in the `Ca_subset` and `Cb_subset` functions.
- [x] Added Py-side documentation to `get_opdm` and `get_tpdm`

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: psi4/src/export_wavefunction.cc [hunk 1]
```diff
@@ -415,8 +417,46 @@ void export_wavefunction(py::module& m) {
...
+                  Return the alpha density matrix, the beta density matrix, or their sum?
+              full_space : bool
+                  Return a density matrix in the space of all orbitals (true) or the active orbitals (false)?
+
+              Returns
+              -------
+              Matrix
+                  The selected one-particle density/transition matrix with Pitzer-ordered orbitals.
+                  Irrep h of the matrix corresponds to orbitals of irrep h.
+                  Element pq is <ψ|a^p a_q|ψ>.
```
Comment (loriab) [inline psi4/src/export_wavefunction.cc:441]: sphinx used to throw a fit if it got non-ascii. but if that hasn't changed, I can patch it up in upcoming docs PR.

File: tests/scf-cholesky-basis/input.dat [hunk 2]
```diff
@@ -19,9 +19,9 @@ e_canonical   = -230.64152543331906
 tol_4 = 3
 tol_5 = 4
 tol_6 = 5
-tol_7 = 8
-tol_8 = 8
-tol_9 = 8
+tol_7 = 7
+tol_8 = 7
+tol_9 = 7
```
Comment (loriab) [inline tests/scf-cholesky-basis/input.dat:24]: oh, wasn't the plan to try `2.e-8`?

Comment (JonathonMisiewicz) [reply tests/scf-cholesky-basis/input.dat:24]: The first time you said anything about that was _after_ my last commit.

File: psi4/src/export_wavefunction.cc (hunk 1 above)
Comment (JonathonMisiewicz) [reply psi4/src/export_wavefunction.cc:441]: I didn't actually bother to check... If you'll handle this during your docs PR, great.

File: tests/scf-cholesky-basis/input.dat (hunk 2 above)
Comment (loriab) [reply tests/scf-cholesky-basis/input.dat:24]: sorry, you were ahead of me :-) I do think it'd be better to stay conservative, but we can patch it up in another PR and save cycles. Whichever you prefer.

Comment (JonathonMisiewicz) [reply tests/scf-cholesky-basis/input.dat:24]: I agree that conservative is better; I just was used to us sticking to `1e-X` checks.

You patching it up in upcoming docs PR to save cycles sounds good, if you're willing.

**Pull request merged**